docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -88,14 +88,6 @@ Deprecated `ezplatform.ee.installer.class` DIC parameter has been removed.
 
 See [eZ Platform v3.0 project update instructions](./ez_platform_v3.0_project_update.md#custom-installers) for upgrade details.
 
-## date-based-publisher
-
-No deprecations or backward compatibility breaks to document.
```

Comment:
Are you sure? POV ping @Nattfarinn @michal-myszka. Not referring to DB schema though, it's covered elsewhere, but AFAIR entire API and SPI layer has been reworked.